.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null

```

Comment:
Remove unrelated files

Reply:
or added .vs/ to .gitignore so this content doesn't get staged.